frontend/src/App.js

```diff
@@ -1,13 +1,41 @@
 import './App.css';
 import LandingPage from './components/LandingPage/LandingPage'
 import MyNavbar from './components/MyNavbar/MyNavbar';
+import React from 'react';
+import { Jumbotron } from 'reactstrap';
+
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from 'react-router-dom';
+
 function App() {
   return (
     <>
       <MyNavbar/>
-      <LandingPage/>
+      <Router>
+          <Switch>
+            <Route path="/" exact component={LandingPage} />
+            <Route path="/test" component={Recipes} />
+          </Switch>
+      </Router>
```

Comment:
I think /test could be changed to something more relevant, both here and in MyNavbar. Maybe something like /FindRecipe?

Reply:
Thanks for the feedback! I changed /test to /findRecipe in App.js and in MyNavbar.js.